### ISSUE TITLE
[v2] Focus Indicator Color over Texture Class

### ISSUE THREAD

**Issue by isralduke** [la-ots/pelican#491]
The *Focus indicator color* on interactive items isn’t easily visible when the item is within a parent with a class of `.bg.texture-accent` . The focus indicator color should be adjusted to fix this.

![CleanShot 2023-09-25 at 09 33 33](https://github.com/la-ots/pelican/assets/10730801/438cee4c-ec82-4008-b9b3-158b20f90c6e)
